### PR TITLE
Migrated to swift 3.0; Added CHANGELOG.md;

### PR DESCRIPTION
**Pull request by danielablg**

### CODE REVIEW

**Review by lucianomarisi — CHANGES_REQUESTED** [review]
Can the code in the README file use the latest API changes?

**Comment by danielablg** [comment]
@lucianomarisi fixed, good catch, thanks

File: README.md [hunk 1]
```diff
@@ -34,20 +34,20 @@ Currently all the most common layout tasks can be completed with SwiftLayout.
 Position:
 
 ```swift
-view.pin(.Top, toEdge: .Top, ofView: view.superview) // default margin 0
-view.pin(.Top, toEdge: .Top, ofView: view.superview, margin: 15)
-view.pin([.Left, .Right], ofView: view.superview) // default margins (0, 0, 0, 0)
-view.pin([.Left, .Right], ofView: view.superview, margins: EdgeMargins(top: 0, left: 15, bottom: 0, right: 15))
+view.pin(edge: .Top, toEdge: .Top, ofView: view.superview) // default margin 0
```
Comment (marius-serban) [inline README.md]: shouldn't enum cases like `.Top` start with lower case letter?

Comment (danielablg) [reply README.md]: fixed

**Review by lucianomarisi — APPROVED** [review]

File: Sources/Constraint.swift [hunk 2]
```diff
@@ -226,7 +226,7 @@ public struct Constraint: ConstraintDefinition {
   public internal(set) var firstAttribute: NSLayoutAttribute
   public internal(set) var secondAttribute: NSLayoutAttribute
   
-  private var _enabled = true
+  fileprivate var _enabled = true
```
Comment (samdods) [inline Sources/Constraint.swift]: should this still be private?

File: Sources/Constraint.swift [hunk 3]
```diff
@@ -250,15 +250,15 @@ public struct Constraint: ConstraintDefinition {
   
   public weak var secondView: View?
   
-  private weak var _constraint: NSLayoutConstraint?
+  fileprivate weak var _constraint: NSLayoutConstraint?
```
Comment (samdods) [inline Sources/Constraint.swift]: private?

Comment (danielablg) [reply Sources/Constraint.swift]: fixed

File: Sources/Constraint.swift (hunk 2 above)
Comment (danielablg) [reply Sources/Constraint.swift]: Indeed they are being used only in the lexical scope they are declared in. Fixed

**Pull request merged**